auth.go

```diff
@@ -56,6 +56,25 @@ func (a *AuthService) login(path string, req *AuthenticateRequest) (*Authenticat
 	return res, nil
 }
 
+// Logout attempts to log a user in with an Auth0 ID token, returning either a (CCloud) token or an error.
+func (a *AuthService) Logout(req *AuthenticateRequest) (*AuthenticateReply, error) {
+	return a.logout("/api/sessions", req)
+}
+
+func (a *AuthService) OktaLogout(req *AuthenticateRequest) (*AuthenticateReply, error) {
```

Comment:
Is this for SSO?

Reply:
Yeah, it looks like there's also a route for SSO: https://github.com/confluentinc/cc-gateway-service/blob/0b88d5d9b0015a0d7e91d6869dab330eeb055125/http/proxy_cloudv1_routes.go#L146